openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activated.json

```diff
@@ -0,0 +1,35 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
```

Comment:
`Activated` is a supported action of `AssignableEvent` so shouldn't we use that value here?

Reply:
have been discussed with you.